wrappers/clang-target-wrapper.c

```diff
@@ -314,6 +315,7 @@ int _tmain(int argc, TCHAR* argv[]) {
         exec_argv[arg++] = _T("-DUNICODE");
         // add the minimum runtime to use for WinRT targets
         exec_argv[arg++] = _T("-Wl,-lmincore");
+        exec_argv[arg++] = _T("-Wl,-lmsvcp120_app");
```

Comment:
LGTM.

Just FYI, I believe this file isn't available for aarch64 though (at least I don't think it's available in mingw-w64 for aarch64), so I believe I'll add an exception to run-tests.sh in the form `if [ "$TEST_DIR" = "aarch64-winrt" ]; continue`, since that combination (aarch64 targeting win8.1) is nonsensical. And with that in mind, it's also probably no point in installing the wrapper tools for that combo. But I can take care of that after merging the rest.

Reply:
Shouldn't this one be `msvcr120_app`, with an `r`, not `msvcp120_app` with a `p`?

Reply:
Indeed. Also in the packaged build I used, mincore is missing for aarch64.

`msvcr120_app` is for C and `msvcr120_app` is for C++. So maybe we need both ? There doesn't seem to be a C++ variant for `vcruntime140_app`. So at least I need to change the 120 one to call the C one.

Reply:
Ah, right. Mincore should be available for aarch64 in the latest upstream mingw-w64 now though.

No, we don't need the C++ libs, we have libstdc++/libc++ instead. (They are bundled or statically linked, so they are no restricted api.) The MSVC C++ ABI is totally incompatible with the itanium C++ ABI used within mingw.

Reply:
FWIW, I reliazed this part is a bit harder than just this; if the headers default to building for UCRT, we need to set `-D__MSVCRT_VERSION__=0x1200` or something similar when building code that is going to use `msvcr120_app`. That's not hard of course, but then the whole libc++ should also have been built with that in mind. (Switching between CRT versions doesn't work for generic precompiled static libraries, which libc++ is in this case.)

Also, arm32 doesn't have msvcr120_app in mingw-w64 either. (There's a videolan patch for adding it for arm32 but it hasn't been upstreamed.)

I would maybe suggest leaving out this part, at least the msvcr120_app part, for now, so we don't hold up all the rest of it due to it, maybe leave it as a todo comment here.